### ISSUE TITLE
command not found when using the always activation mode

### ISSUE THREAD

**Issue by santagada** [stef-levesque/vscode-perforce#136]
Hi,

The perforce extension used to work for opening just one file until the update to have different activation modes. Even setting it to Always I still get command not found on all perforce commands I try. I have:

    "perforce.activationMode": "always",

on my settings and I didn't see any error on the developer console (except when trying to run a command). Why isn't the extension loading?

**Comment by jel-massih** [comment]
Yeah I am able to replicate this (and have ran into this several times). I will see if I can scrounge together some time to look into this.
